### ISSUE TITLE
LPR improvements

### ISSUE THREAD

**Issue by tarcadius** [IM3S/LDAR_Sim#30]
Should we be moving away from having a single LPR and build in capacity to assign different LPRs to different types of equipment or facilities based on historical data? LPR could even sample from a distribution. Or should we just leave it as a single value?

@tbarchyn @KeeganShaw-GIS @AkaMrmuffin @c-vollrath

**Comment by c-vollrath** [comment]
Yes to moving away from having a single LPR value and disaggregating this down to the equipment scale. Would it work to divide a site-level LPR by the number of equipment(s) on site depending on the inputs provided by the user or a default distribution in LDAR-Sim? Or would it have to be proportional based on some pieces of equipment being more likely to have leaks arise than others? If the latter, how would we approach this? The distribution could work well here!

**Comment by tarcadius** [comment]
To continue following the advice Tom keeps giving, if we build in equipment-scale LPR functionality it needs to be an enhancement - an option - and the simpler version still needs to exist. Ideally operators have sufficient data to inform equipment scale LPR. However, this data might rarely be available.

@c-vollrath Yes, the simplest approach would be to add up all the LPRs for all sites, add up all the equipments, and allocate accordingly. That doesn't take into account BMPs, equipment type, age, etc. etc. So, start simple...

Another way to do it would be to switch things around - rather than generate leaks AT each facility, generate leaks all together and then assign them to facilities and equipment and even components. If all of the leaks in your input distrubution have a facility type, equipment type, and component type associated with them, then this is easy. Otherwise, good luck!

I want to come back to this one as it is one of the more complicated options but definitely important... I think start easy as we have with other topics, perhaps by allocating using the first method you described Coleman...
